server/src/test/java/org/elasticsearch/index/mapper/WholeNumberFieldMapperTests.java

```diff
@@ -92,6 +92,19 @@ public void testMetricAndDimension() {
         );
     }
 
+    public void testSingleValuedField() throws IOException {
```

Comment:
Is there any reason why this test is in `WholeNumberFieldMapperTests` and not in `NumberFieldMapperTests`?